domains/faul.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/KhongPhaiHoangQuangHuy/FaulWeb.git",
+    "owner": {
+        "username": "KhongPhaiHoangQuangHuy",
+        "email": "khongphaihoangquanghuy@gmail.com",
+        "twitter": "FaulVerlierer"
+    },
+    "record": {
+        "CNAME": "https://khongphaihoangquanghuy.github.io/FaulWeb/"
```

Comment:
Invalid CNAME. CNAME must be a hostname with not https:// or /. Also, your site seems to be an endless redirect, please double check that it works.